src/handlers/modal-submit.js

```diff
@@ -0,0 +1,51 @@
+const { formatPRMessage } = require('../utils/pr-formatter');
+
+async function handleModalSubmit(interaction) {
+  if (interaction.customId.startsWith('pr-template-modal-')) {
+    const channelId = interaction.customId.replace('pr-template-modal-', '');
+    const prUrl = interaction.fields.getTextInputValue('urlInput');
+    const title = interaction.fields.getTextInputValue('titleInput');
+    const overview = interaction.fields.getTextInputValue('overviewInput');
+    const howToTest = interaction.fields.getTextInputValue('howToTestInput');
+
+    const formattedMessage = formatPRMessage({
+      prUrl,
+      title,
+      overview,
+      howToTest,
+      requester: interaction.user,
+    });
+
+    try {
+      const channel = await interaction.client.channels.fetch(channelId);
+
+      if (!channel) {
+        throw new Error('Channel not found');
+      }
```

Comment:
Here is better display the error to the user, maybe with interaction.reply({ content: "sdfkjasd", ephemeral: true })